poc_iot_injector/src/server.rs

```diff
@@ -70,6 +70,7 @@ impl Server {
     }
 
     async fn handle_poc_tick(&mut self) -> Result {
+        metrics::increment_counter!("poc_iot_injector_server_tick_count");
```

Comment:
what's the utility of this measurement? this seems like it's just measuring rust's ability to loop on a duration?

Reply:
Agreed. Removing this one.